content/docs/practices/naming.md

```diff
@@ -7,41 +7,43 @@ sort_rank: 1
 
 A metric name...
 
-* ...must comply with the [data model](/docs/concepts/data_model/#metric-names-and-labels) for valid characters.
-* ...should have a (single-word) application prefix relevant to the domain the
+- ...must comply with the [data model](/docs/concepts/data_model/#metric-names-and-labels) for valid characters.
+- ...should have a (single-word) application prefix relevant to the domain the
   metric belongs to. The prefix is sometimes referred to as `namespace` by
   client libraries. For metrics specific to an application, the prefix is
   usually the application name itself. Sometimes, however, metrics are more
   generic, like standardized metrics exported by client libraries. Examples:
- * <code><b>prometheus</b>\_notifications\_total</code>
-   (specific to the Prometheus server)
- * <code><b>process</b>\_cpu\_seconds\_total</code>
-   (exported by many client libraries)
- * <code><b>http</b>\_request\_duration\_seconds</code>
-   (for all HTTP requests)
-* ...must have a single unit (i.e. do not mix seconds with milliseconds, or seconds with bytes).
-* ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers). See below for a list of base units.
-* ...should have a suffix describing the unit, in plural form. Note that an accumulating count has `total` as a suffix, in addition to the unit if applicable.
- * <code>http\_request\_duration\_<b>seconds</b></code>
- * <code>node\_memory\_usage\_<b>bytes</b></code>
- * <code>http\_requests\_<b>total</b></code>
-   (for a unit-less accumulating count)
- * <code>process\_cpu\_<b>seconds\_total</b></code>
-   (for an accumulating count with unit)
- * <code>foobar_build<b>\_info</b></code>
-   (for a pseudo-metric that provides [metadata](https://www.robustperception.io/exposing-the-software-version-to-prometheus) about the running binary)
-* ...should represent the same logical thing-being-measured across all label
+- <code><b>prometheus</b>\_notifications_total</code>
+  (specific to the Prometheus server)
+- <code><b>process</b>\_cpu_seconds_total</code>
+  (exported by many client libraries)
+- <code><b>http</b>\_request_duration_seconds</code>
+  (for all HTTP requests)
+- ...must have a single unit (i.e. do not mix seconds with milliseconds, or seconds with bytes).
+- ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers). See below for a list of base units.
+- ...should have a suffix describing the unit, in plural form. Note that an accumulating count has `total` as a suffix, in addition to the unit if applicable.
+- <code>http_request_duration\_<b>seconds</b></code>
+- <code>node_memory_usage\_<b>bytes</b></code>
+- <code>http_requests\_<b>total</b></code>
+  (for a unit-less accumulating count)
+- <code>process_cpu\_<b>seconds_total</b></code>
+  (for an accumulating count with unit)
+- <code>record_processed\_<b>latest_unix_time</b></code>
+  (for a timestamp that tracks a latest occurence)
```

Comment:
here is the new entry,

debate open as to wether to standardize on `_timestamp_seconds` or `_unix_time`

unix time does imply seconds https://en.wikipedia.org/wiki/Unix_time
though the former is more explicit